Library/Formula/fcitx-remote-for-osx.rb

```diff
@@ -0,0 +1,31 @@
+class FcitxRemoteForOsx < Formula
+  desc "handle input method in command-line"
+  homepage "https://github.com/CodeFalling/fcitx-remote-for-osx"
+  url "https://github.com/CodeFalling/fcitx-remote-for-osx/archive/0.0.2.tar.gz"
+  sha256 "746505694090d98d5ad572758e7597c49d3bea1b00208a8272c3648aad15ad69"
+
+  INPUT_METHOD = %w[baidu-pinyin baidu-wubi sogou-pinyin qq-wubi squirrel-rime osx-pinyin]
+  INPUT_METHOD.each do |im|
```

Comment:
Please spell these out fully rather than dynamically. Also, is there any reason to not just build them all by default?